Gemfile

```diff
@@ -12,6 +12,8 @@ group :development do
   gem "standard", "~> 1.3"
   gem "get_process_mem"
   gem "ruby-lsp", require: false
+  gem "yard", require: false
+  gem "yard-rustdoc", "~> 0.1", require: false
```

Comment:
Not requiring yard because it has a few patches / global additions, we don't need those.